domain-ee/ee-ep-merge-app/integration/test_merge_request.py

```diff
@@ -94,12 +94,18 @@ async def test_completed_success_with_duplicate_contention(
             assert_successful_response(response)
 
     @pytest.mark.asyncio(scope="session")
-    async def test_completed_no_ep400_contentions(
-        self, get_claim_endpoint: MqEndpoint, get_claim_contentions_endpoint: MqEndpoint, put_tsoj_endpoint: MqEndpoint, cancel_claim_endpoint: MqEndpoint
+    async def test_completed_no_ep400_contentions_on_first_attempt(
+        self,
+        get_claim_endpoint: MqEndpoint,
+        get_claim_contentions_endpoint: MqEndpoint,
+        put_tsoj_endpoint: MqEndpoint,
+        create_claim_contentions_endpoint: MqEndpoint,
+        cancel_claim_endpoint: MqEndpoint,
     ):
         get_claim_endpoint.set_responses([pending_claim_200])
-        get_claim_contentions_endpoint.set_responses([pending_contentions_200, response_204])
+        get_claim_contentions_endpoint.set_responses([pending_contentions_200, response_204, ep400_contentions_200])
```

Comment:
Such a neat way to update this integration test.

Is there still a test for the original scenario, when no contentions ever get created? That still seems valuable to have.

Reply:
I added one back in there and it made me realize a potential bug in the MqEndpoint.py when adding responses. 👍